packages/react/src/themes/teams/components/List/listItemStyles.ts

```diff
@@ -100,6 +100,7 @@ const listItemStyles: ComponentSlotStylesInput<ListItemPropsAndState, any> = {
   }),
   endMedia: ({ props: p }) => ({
     ...(p.selectable && { display: 'none' }),
+    flexShrink: 0,
```

Comment:
I'm still not sure if the List behaves correctly... Pls. take a look at this screenshot:

![Screen Recording 2019-06-04 at 14 21 49](https://user-images.githubusercontent.com/5442794/58878782-5e9c4880-86d4-11e9-9895-bba9270a9489.gif)

here is the example I used:

```jsx
import * as React from 'react'
import * as _ from 'lodash'
import { List, Button } from '@stardust-ui/react'

const items = [
  {
    key: 'ftp',
    endMedia: {
      content: (
        <Button.Group
          buttons={_.times(20).map(i => ({
            key: 'accept' + i,
            icon: 'accept',
            iconOnly: true,
            text: true,
          }))}
        />
      ),
      styles: { flexShrink: 0 },
    },
    content:
      'Use the online FTP application to input the multi-byte application!Use the online FTP application to input the multi-byte application!',
  },
]

const ListExample = () => <List items={items} selectable />

export default ListExample
```

Reply:
The `media` and `endMedia` should have width as the content inside them. The `content` between them is suppose to grow in order to fill the list item. If we follow this, I think your example is still correct.